src/Workspaces/Core/Portable/Editing/ImportAdderService.cs

```diff
@@ -121,12 +121,14 @@ private static SyntaxNode GenerateNamespaceImportDeclaration(INamespaceSymbol na
             var model = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
             var nodesWithExplicitNamespaces = syntaxNodes
-                .Select(n => (syntaxnode: n, namespaceSymbol: GetExplicitNamespaceSymbol(n, model)))
-                .Where(x => x.namespaceSymbol != null);
+                .Select(n => (syntaxnode: n, namespaceSymbol: GetExplicitNamespaceSymbol(n, model)));
 
             var addedSymbols = new HashSet<INamespaceSymbol>();
             foreach (var (node, namespaceSymbol) in nodesWithExplicitNamespaces)
             {
+                if (namespaceSymbol is null)
```

Comment:
We used to incorrectly assume that the 'namespaceSymbol' was not-null at the beginning of the loop. Now that we fixed nullable deconstruction analysis we have to null check it in a way the compiler understands.